### ISSUE TITLE
Decision Proposal 257 - Customer Data Payloads for Telco

### ISSUE THREAD

**Issue by CDR-API-Stream** [ConsumerDataStandardsAustralia/standards#257]
This decision proposal contains a recommendation for the candidate URIs and payloads for the Customer data cluster for the telecommunications sector.

The decision proposal is embedded below:
[Decision Proposal 257 - Customer Data Payloads for Telco.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/9238278/Decision.Proposal.257.-.Customer.Data.Payloads.for.Telco.pdf)

This consultation will be open for feedback until the 12th August 2022.

-----
The noting paper was updated to correct a type on the 2nd August.  The original version of the not paper can be found [here](https://github.com/ConsumerDataStandardsAustralia/standards/files/8996244/Decision.Proposal.257.-.Customer.Data.Payloads.for.Telco.pdf)

**Comment by LeanneVOD** [comment]
The reference to NMI Standards (at the top of page 3) appears to be from a previous document relating to energy. Assume this reference should be to customer data.

**Comment by CDR-API-Stream** [comment]
> The reference to NMI Standards (at the top of page 3) appears to be from a previous document relating to energy. Assume this reference should be to customer data.

Thank you @LeanneVOD for pointing out this error (which we should have picked up in review).  We have updated the Decision Proposal to fix this error.

**Comment by cybertrekman** [comment]
TPG Telecom have an issue with the approach taken to create standards before the rules are known. We would expect that to have a meaningful dialogue regarding the standard requires rules to be in place otherwise comments on the standards are rather pointless. The telco market is quite unlike banking or energy, yes we have customers and authorised rep's, but we also have arrangements for sharing the product purchased between parties that may only be loosely associated, for example; a customer can purchase a product from one telco that allows donation of unused available data to anyone with service with the same service provider. There should be no expectation that a customer sharing their spare data allowance with another customer enables either party to view information relating to either party. Telco's regularly have special offers and plans that provide free use of the service for a short period, and/or half price or double allowances for use for a specified period. Telco's have a range of plans that result in some customers having multiple services (fixed, local, broadband) on a single account, or on multiple accounts, the product construct is based on the service (Note: document 256 seems to have this switched about), where a product set then depends upon whether the service is paid as a post paid, pre-paid or hybrid of both (i.e. some parts of the service paid in advance and usage assessed at the end of a set period) and the product is the amount paid for the service at a set interval that may be 7 days, 28 days, quarterly, bi-annual or annual. Generally customers are not locked into plans (product) and this can vary month to month e.g. a customer may know they are going to need more capacity in a given month and may choose to upgrade their plan for that month, or the opposite. Customers can also readily change the interval period and product at any time, for example a customer may be on a 7 day plan to trial a service and then switch to a monthly or longer interval plan, or a customer on a long payment interval plan may choose to switch to a shorter duration plan at any time before expiry of the plan. We are therefore unclear how this relates to the expectations of the CDR scheme and how this relates to the standards. Also, large corporate and government entities have unique plans to suit their needs and CDR will have no benefit to this category of consumer but the effort required to incorporate this category of customers into CDR will be quite significant. 
Telco's activities are heavily regulated including the sharing of customer data into the Integrated Public Number Database (IPND) which is a record of every active service that uses a public number. If we are to adopt a data standard it should at least be consistent with our existing obligations to provide data to the IPND. 
[IPND Data Users and Data Providers Technical Requirements 20211117.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/9246347/IPND.Data.Users.and.Data.Providers.Technical.Requirements.20211117.pdf)
[G619_2007.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/9246349/G619_2007.pdf)

**Comment by CDR-API-Stream** [comment]
Hi @jpopovaclark, thanks for your feedback.  Here is our response:

> Hi All. Its great to be joining this community and would like to wholeheartedly endorse this approach to fostering substantive whole-of-sector cooperation, particularly when generating technical standards. I hope the innovativeness of this approach is recognised and appreciated by senior government stakeholders.

Thank you.  Encouragement is always welcome, as are any suggestions for improvement.  Sometimes improvements can be more easily identified by people coming into the process cold than those of us that have been doing this for a while.

> I'm from **Optus** and we would just like to advise that we at Optus believe that meaningful contribution to these standards will not be of value until the Telco Rules are finalised (or at least fundamentally close to finalised). For instance, we are unclear if enterprise customers are in scope, whether prepaid products are included, which telecommunication products & services are in scope nor how products bundled with out-of-scope services will be treated. In just these examples alone, the appropriateness of the detail of this proposal cannot be meaningfully assessed.

We understand the perspective that the standards can only ever be in draft until the rules are defined.  It is worth noting, however, that we have repeatedly found that consulting on the two in parallel has significant advantages.  It is often the case that the standards consultations raise issues that end up being addressed in the rules and vice versa.  We would encourage you to reconsider your position as we would very much like to get feedback from Optus in the process.

Regarding the examples you give for instance, even if we assume that C&I customers will not be in scope there is still value in reviewing the customer payloads from a consumer perspective as consumers will clearly be in scope based on the designation instrument alone.

Equally, while we don't know the scope of products and services, the reality is that getting to the heart of the products, services and bundles that should be included depends on an understanding of what they are and where the complexity lies.  Feedback on these issues in our technical consultations will be of immense value to the rules team in developing the rules.

Some of these concerns are addressed in [Noting Paper 255](https://github.com/ConsumerDataStandardsAustralia/standards/issues/255) that we published once the designation instrument was made.  You may get value from reviewing that document.

**Comment by CDR-API-Stream** [comment]
Thanks for your feedback @cybertrekman.  Here is our response:

> TPG Telecom have an issue with the approach taken to create standards before the rules are known. We would expect that to have a meaningful dialogue regarding the standard requires rules to be in place otherwise comments on the standards are rather pointless.

Please refer to our response to Optus above on this topic.  Obviously the DSB holds a different position on this issue based on our experience to date in developing standards.

> The telco market is quite unlike banking or energy, yes we have customers and authorised rep's, but we also have arrangements for sharing the product purchased between parties that may only be loosely associated, for example; a customer can purchase a product from one telco that allows donation of unused available data to anyone with service with the same service provider. There should be no expectation that a customer sharing their spare data allowance with another customer enables either party to view information relating to either party.

It is exactly these sectoral idiosyncrasies that will be drawn out via the technical consultations that may then result in feedback to the rules before they are finalised.  It is to uncover issues of this nature that we do the consultations in parallel.

> Telco's regularly have special offers and plans that provide free use of the service for a short period, and/or half price or double allowances for use for a specified period. Telco's have a range of plans that result in some customers having multiple services (fixed, local, broadband) on a single account, or on multiple accounts, the product construct is based on the service (Note: document 256 seems to have this switched about), where a product set then depends upon whether the service is paid as a post paid, pre-paid or hybrid of both (i.e. some parts of the service paid in advance and usage assessed at the end of a set period) and the product is the amount paid for the service at a set interval that may be 7 days, 28 days, quarterly, bi-annual or annual. Generally customers are not locked into plans (product) and this can vary month to month e.g. a customer may know they are going to need more capacity in a given month and may choose to upgrade their plan for that month, or the opposite. Customers can also readily change the interval period and product at any time, for example a customer may be on a 7 day plan to trial a service and then switch to a monthly or longer interval plan, or a customer on a long payment interval plan may choose to switch to a shorter duration plan at any time before expiry of the plan. We are therefore unclear how this relates to the expectations of the CDR scheme and how this relates to the standards. Also, large corporate and government entities have unique plans to suit their needs and CDR will have no benefit to this category of consumer but the effort required to incorporate this category of customers into CDR will be quite significant.

Most of this feedback would actually be relevant to the product/service and account data clusters rather than the customer data cluster.  The customer resource APIs are intended to convey the information held by a data holder about the customer.  Things like name, address, etc.  This can be reviewed and assessed without needing to reference the products or services they are currently engaged with.

This is an explicit design feature of the CDR due to the strong consent model inherent.  It is entirely within the scope of a customer to share only their customer details with an ADR (for instance to pre-pop and application form) without consenting to the sharing of any account, balance or billing information.

> Telco's activities are heavily regulated including the sharing of customer data into the Integrated Public Number Database (IPND) which is a record of every active service that uses a public number. If we are to adopt a data standard it should at least be consistent with our existing obligations to provide data to the IPND. [IPND Data Users and Data Providers Technical Requirements 20211117.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/9246347/IPND.Data.Users.and.Data.Providers.Technical.Requirements.20211117.pdf) [G619_2007.pdf](https://github.com/ConsumerDataStandardsAustralia/standards/files/9246349/G619_2007.pdf)

The data holders that are already incorporated in the CDR in banking and energy would argue they are also pretty heavily regulated. This is not a new concept for the CDR but the industry specifics are worth highlighting a drawing out so they can be appropriately addressed.

Thank you for highlight the IPND reference.  This will be of value when we develop the consultation on account data and product data (and highlights the value of consulting early).

**Comment by CDR-API-Stream** [comment]
Thanks everyone for the helpful feedback. The feedback will be incorporated and draft standards will be published representing the progressive position of the evolving Telco standards in due course.

This consultation will now be closed.
